### ISSUE TITLE
After plex restart, "Most Concurrent Streams" skyrockets. 

### ISSUE THREAD

**Issue by zetas** [Tautulli/Tautulli#867]
**Version:** 1.4.13

**Branch:** master

**Commit hash:** 1a2e205c1f0ab91582ce498a8828ee07f573cd4f

**Operating system:** Ubuntu 16.04

**Python version:** 2.7.12 (default, Jul 1 2016, 15:12:24) [GCC 5.4.0 20160609]

**What you did?** Stop plex for awhile and start it.

**What happened?** Concurrent streams stat went up to 142.

**What you expected?** It should not have changed at all, it should have stayed at 3

**How can we reproduce your issue?**

I'm not sure, stop plex for awhile and leave plexpy running then restart it? I'm unable to reproduce however I this happened once before a few weeks ago, the concurrent streams was in the thousands. I couldn't figure out how to fix it but since It never happened before I figured it was a fluke. I nuked the plexpy database and started fresh. It sucked but I thought it was the only way.

**What are your (relevant) settings?**
None that I can think of. I have IP logging on. Plex is local to plexpy, not using SSL.

**Link to logs:** http://paste.ubuntu.com/23301475/

**Comment by JonnyWong16** [comment]
You have some really weird stuff going on in those logs. You had one session stuck in the database and it kept adding it to your history over a hundred times.

```
2016-10-09 15:28:44 - DEBUG   :: Thread-12 : PlexPy ActivityProcessor :: Fetching metadata for item ratingKey 17088
```

I'm not sure where it got stuck or how much it wrote to the database. Does this item show up on the history page? If so, you should be able to delete it from the history table. Would you mind sending me your database file via a PM either on Gitter or Discord so I might be able to fix this in the future?

Just incase you didn't know, there is a "Flush Temporary Sessions" under Extra Settings now which fixes things like this. It looks like you did this already according to the last line in your logs.

```
2016-10-09 23:44:57 - DEBUG   :: CP Server Thread-9 : PlexPy Database :: Clearing temporary sessions from database.
```

**Comment by zetas** [comment]
Yea, I dug around in the current closed and open issues for solutions before posting and you suggested the clear temp sessions endpoint to someone else with weird stats so i thought that might help. It actually boosted the concurrent streams from 100 to 142 lol. I'd be happy to share my database file on discord. Feel free to email me the details: zetas@zet.as

**Comment by zetas** [comment]
Oh also, i don't see anything suspicious in the history, nothing like a hundred items repeated.

**Comment by JonnyWong16** [comment]
Please join my discord chat https://discord.gg/36ggawe

**Comment by JonnyWong16** [comment]
Found the problem. Temporarily fixed on `dev` 6aa0d4cd0b8526c5708a99eed3dc86972cbab609.

**Comment by djderan** [comment]
This happened to me as well is there a way to clear this data out without nuking the database?

**Comment by JonnyWong16** [comment]
You can manually delete them from the database if you know how.

Or try this:
1. Enable the API under Settings > Access Control and remember your API key.
2. Shutdown PlexPy and open your config.ini file in a text editor.
3. Set `api_sql =  1` in the config file.
4. Restart PlexPy.
5. Go to these two urls to run the SQL commands through the API. Substitute `YOUR_API_KEY`

```
http://localhost:8181/api/v2?apikey=YOUR_API_KEY&cmd=sql&query=DELETE%20FROM%20session_history%20WHERE%20id%20NOT%20IN%20(SELECT%20id%20FROM%20session_history_metadata)%3B
http://localhost:8181/api/v2?apikey=YOUR_API_KEY&cmd=sql&query=DELETE%20FROM%20session_history_media_info%20WHERE%20id%20NOT%20IN%20(SELECT%20id%20FROM%20session_history_metadata)%3B
```
1. Go back to PlexPy to make sure everything is back to normal.
2. Repeat steps 2-4 and set `api_sql = 0`
3. Disable the API if you want.

**Comment by djderan** [comment]
cool thanks fixed it by running query delete from session_history where id not in (select id from session_history_metadata)

**Comment by JonnyWong16** [comment]
You also need to delete from `session_history_media_info`.

**Comment by djderan** [comment]
sorry I did that as well everything looks right now!

**Comment by JonnyWong16** [comment]
Fixed in v1.4.14.
